src/gui/mod.rs

```diff
@@ -315,7 +323,7 @@ impl Application for Ajour {
         content = content.push(Space::new(Length::Units(0), Length::Units(DEFAULT_PADDING)));
 
         match self.mode {
-            AjourMode::MyAddons => {
+            Mode::MyAddons(flavor) => {
```

Comment:
We now have flavor from the mode we have selected. A bit prettier than asking `config.wow.flavor` very single time.